src/components/Form/bookForm.css

```diff
@@ -1,32 +1,44 @@
 form {
-  display: grid;
-  place-items: center;
-  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  margin-top: 2rem;
+  margin-left: 3%;
+  margin-bottom: 3rem;
 }
 
-.author-1 {
-  border: 2px solid #000;
-  border-radius: 4px;
-  width: 85vw;
-  height: 3rem;
-  font-size: 20px;
-  margin-bottom: 1rem;
+.h2 {
+  font-size: 25px;
+  text-transform: uppercase;
+  color: #c5c5c5;
+  font-weight: 700;
```

Comment:
- [ ] Please 🙏🏽 use the styles assigned to the title in the original design to match with Zeplin design 👇🏽 

![image](https://github.com/Booze33/book-store/assets/58642949/45380c0d-2208-4dc0-8631-8f02753c692c)